git_push/defs.bzl

```diff
@@ -0,0 +1,39 @@
+"git_push rule for delivering to GitHub"
+
+# We can't use the bazel-lib one, because it doesn't have a program to read stamp vars.
+# see https://github.com/aspect-build/rules_js/pull/384#issue-1337742941
+# load("@aspect_bazel_lib//lib:expand_make_vars.bzl", "expand_template")
+# buildifier: disable=bzl-visibility
+load("@aspect_rules_js//js/private:expand_template.bzl", "expand_template")
```

Comment:
lol. my little private utility is getting a lot of public attention

Reply:
yes we really need to write another tiny go program with a toolchain hanging off bazel-lib so we can upstream that